chapters/modules/chapter_2/pages/index.adoc

```diff
@@ -1,18 +1,51 @@
 [id="index"]
 
-= BUIDL and Tooling
+= Smart Contracts in Cairo
```

Comment:
@l-henri @gyan0890 is the index for introducing Chapter 2: Smart Contracts in Cairo.